backend/src/test/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessorTest.java

```diff
@@ -115,15 +115,14 @@ void testCamelEndpointProcessing() {
         JsonObject notifMetadata = payload.getJsonObject(NOTIF_METADATA_KEY);
         assertEquals(properties1.getDisableSslVerification().toString(), notifMetadata.getString("trustAll"));
         assertEquals(properties1.getUrl(), notifMetadata.getString("url"));
-        assertEquals(properties1.getSubType(), notifMetadata.getString("type"));
```

Comment:
todo: still test this one for backwards compatibility

Reply:
todo: needs to be equal to `endpoint1.getSubType()`

Reply:
This check was missing, checking that the "type" prop of notifMetadata is endpoint1.getSubType